ee/clickhouse/queries/test/__snapshots__/test_breakdown_props.ambr

```diff
@@ -43,7 +43,7 @@
      WHERE team_id = 2
        AND event = '$pageview'
        AND timestamp >= toDateTime('2020-01-01 00:00:00')
-       AND timestamp <= toDateTime('2020-01-12 23:59:59')
```

Comment:
if we're going forward a second _on the same day_ should this be `<` and not `<=`

Reply:
Yeah I think that's correct. I keep going back and forth on options... I might try that out